chroma-agent/chroma_agent/action_plugins/manage_targets.py

```diff
@@ -428,7 +428,17 @@ def mount_target(uuid, pacemaker_ha_operation):
     # This is called by the Target RA from corosync
     info = _get_target_config(uuid)
 
-    if agent_result_is_error(import_target(info['device_type'], info['bdev'])):
+    import_retries = 100
+    errored = True
+
+    for i in xrange(import_retries):
+        errored = agent_result_is_error(import_target(info['device_type'], info['bdev'], pacemaker_ha_operation))
```

Comment:
Why does this need to be repeated (without any sleep between attempts even) up to 100(!) times to succeed?